src/sty/arrow.sty

```diff
@@ -1,20 +1,27 @@
-Set x {
```

Comment:
Add repro instructions to PR: run Penrose with sub/angles.sub, sty/arrow.sty, and dsll/linear-algebra.dsl

Reply:
I added reproducible examples to all my shapes, I'll push it soon